README.md

```diff
@@ -1,3 +1,142 @@
 # adage-server
```

Comment:
These reflect @rzelayafavila's changes that got pulled it when I attempted to rebase on master.